webapp/channels/src/components/widgets/inputs/input/input.test.tsx

```diff
@@ -1,28 +1,40 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {shallow} from 'enzyme';
+import {act, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
test migrated from enzyme to testing library 🦾 and added a couple of new tests to cover some uncovered scenarios